From 6bae70f86196b5f8f8767a4e2f3c1b7583459dac Mon Sep 17 00:00:00 2001
From: Hans Rosenfeld <hans.rosenfeld@joyent.com>
Date: Wed, 19 Dec 2018 16:27:28 +0000
Subject: [PATCH] OS-7392 bhyve optstr wrong for -Y Reviewed by: John Levon
 <john.levon@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Reviewed by: Jorge Schrauwen <sjorge@blackdot.be>
 Approved by: John Levon <john.levon@joyent.com>

---
 usr/src/cmd/bhyve/bhyverun.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/usr/src/cmd/bhyve/bhyverun.c b/usr/src/cmd/bhyve/bhyverun.c
index 317d640a2c..405accda35 100644
--- a/usr/src/cmd/bhyve/bhyverun.c
+++ b/usr/src/cmd/bhyve/bhyverun.c
@@ -1024,7 +1024,7 @@ main(int argc, char *argv[])
 #ifdef	__FreeBSD__
 	optstr = "abehuwxACHIPSWYp:g:G:c:s:m:l:B:U:";
 #else
-	optstr = "abehuwxACHIPSWY:g:G:c:s:m:l:B:U:";
+	optstr = "abehuwxACHIPSWYg:G:c:s:m:l:B:U:";
 #endif
 	while ((c = getopt(argc, argv, optstr)) != -1) {
 		switch (c) {
-- 
2.21.0

